spikingjelly/activation_based/triton_kernel/triton_utils.py

```
@@ -139,13 +140,32 @@ def _check_pytorch_version(version_s: str = "2.4") -> bool:
-            pass  # ignore the errors
+    for cache_dir in _triton_codegen_cache_dirs():
+        if not cache_dir.is_dir():
+            continue
+        for pattern in ("*.py", "*.py.tmp", "*.tmp"):
+            for f in cache_dir.glob(pattern):
+                try:
+                    f.unlink()
+                except FileNotFoundError:
+                    pass
+                except BaseException:
+                    pass  # ignore the errors
```

Ruff (0.15.11)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)